addons/project/project.py

```diff
@@ -60,10 +60,10 @@ class project_task_type(osv.osv):
                                'there are no records in that stage to display.'),
     }
 
-    def _get_default_project_ids(self, cr, uid, ctx={}):
-        project_id = self.pool['project.task']._get_default_project_id(cr, uid, context=ctx)
-        if project_id:
-            return [project_id]
+    def _get_default_project_ids(self, cr, uid, ctx=None):
+        ctx = dict(ctx or {})
```

Comment:
@rim-odoo : `ctx = dict(ctx or {})` and  `if cxt is None: ctx = {}` they are retrun same value.